.github/workflows/test_policy_sql.yml

```diff
@@ -20,13 +20,11 @@ jobs:
           "plugins/source/azure",
           "plugins/source/gcp",
         ]
-        dbversion: [ "postgres:10" ]
         go: [ "1.18" ]
-        platform: [ ubuntu-latest ] # can not run in macOS and widnowsOS
-    runs-on: ${{ matrix.platform }}
+    runs-on: ubuntu-latest # can not run in macOS and widnowsOS
     services:
       postgres:
-        image: ${{ matrix.dbversion }}
+        image: postgres:11
```

Comment:
Moved this out of the matrix so renovate can update it, see https://github.com/cloudquery/cloudquery/pull/1294/files#diff-bd076b414d874e4ac3b550c05f66e1fdfbfb5f549c6f5ed851bdb85bad42b91eR34